### DISCUSSION TITLE
Issues with DAKOTA installation

### DISCUSSION THREAD

**Discussion by GianmarcoFarro** [community/community#57313] [Programming Help]
### Select Topic Area

Question

### Select Feature Area

- [ ] Issues
- [ ] Projects

### Body

I am working on Ubuntu 18.04.06 and installing DAKOTA to run UQ analysis. Following the INSTALL file provided with the unzipped package, when running     sudo make     I get the following error:

[ 30%] Generating ../generated/src/NIDR_keywds0.h

../packages/external/nidr/nidrgen:
	botch on line 1 of "/usr/local/dakota-6.18.0-public-src-cli/src/dakota.input.nspec":
" unexpected.
src/CMakeFiles/dakota_order_input.dir/build.make:74: recipe for target 'generated/src/NIDR_keywds0.h' failed
make[2]: *** [generated/src/NIDR_keywds0.h] Error 1
make[2]: *** Deleting file 'generated/src/NIDR_keywds0.h'
CMakeFiles/Makefile2:6704: recipe for target 'src/CMakeFiles/dakota_order_input.dir/all' failed
make[1]: *** [src/CMakeFiles/dakota_order_input.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....

The line the error message refers to in the specified path is the following:

KEYWORD01 environment {NIDRProblemDescDB::env_start}

Could you suggest me how to fix the problem?

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by tuves** [comment]
Hi @GianmarcoFarro,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project, such as the [Dakota](https://github.com/bfovet/dakota/blob/master/INSTALL). It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
